release-3.7.0-announcement.md

```diff
@@ -0,0 +1,31 @@
+We're excited to announce the availability of Jamulus 3.7.0!
+
+This release re-organizes Server lists to make room for more servers:
+- Default has been renamed to Any Genre 1.
+- Any Genre has been renamed to Any Genre 2.
+- Any Genre 3 has been added.
+- Choral/Barbershop has been added.
+- Classical/Folk/Choral has been renamed to Classical/Folk.
+
+Besides that, tens of usability improvements, enhancements, bugfixes and optimizations have been integrated.
```

Comment:
Are you sure about

> tens of

?

Reply:
Do you mean the actual number or the wording? I'm counting 25-30 Changelog entries, so I'd say the number is OK. But I'm open to change this :)

Reply:
Maybe numerous or something similar would be better.

Reply:
Thanks, done.